app/views/subscriptions/edit_search_criteria.html.haml

```diff
@@ -0,0 +1,19 @@
+- content_for :page_title_prefix, t('.title')
```

Comment:
edit_search_criteria.html.haml is a partial right? so should it be named _blahblah?

Reply:
it is a view, not a partial